hermes/probe/delete/delete_test.go

```diff
@@ -114,13 +114,14 @@ func genTestTarget(cfg *monitorpb.HermesProbeDef, t *testing.T) *probe.Target {
 	}
 }
 
-// setupFakeSystem sets up the fake storage system through the fake client.
-func setupFakeSystem(ctx context.Context, t *testing.T) stiface.Client {
+// fakeStorageClient sets up the fake storage system through the fake client
```

Comment:
Nit: This is not technically correct. The fake client implements a fake client, not a fake storage system (which is the server-side).
If the fake implemented the storage system, we could use a real client (this would actually be preferred).

Reply:
Ah okay, I've removed this comment.